### PR TITLE
Added moon phase module

### PR DESCRIPTION
**Pull request by Elder-of-Ozone**

### CODE REVIEW

File: i3pystatus/moon.py [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+                                Last Quarter:
+                                Waning Crescent:
+                         }
+        """
+
+        settings = (
+            "format",
+            ("status", "Current moon phase"),
+            ("illum", "Percentage that is illuminated"),
+            ("color", "Set color"),
```
Comment (enkore) [inline i3pystatus/moon.py]: I think you mean `phase_color` here?

File: i3pystatus/moon.py [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+            "First Quarter": "#265ECC",
+            "Waxing Gibbous": "#392FBF",
+            "Full Moon": "#4C00B3",
+            "Waning Gibbous": "#871181",
+            "Last Quarter": "#C32250",
+            "Waning Crescent": "#FF341F",
+        }
+
+        def pos(now=None):
+            now = None
```
Comment (enkore) [inline i3pystatus/moon.py]: now = None makes the parameter unnecessary

File: i3pystatus/moon.py [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            days_in_second = 86400
+
+            if now is None:
+                now = datetime.datetime.now()
+
```
Comment (enkore) [inline i3pystatus/moon.py]: And the if here, too, since it is always None at this point

Comment (Elder-of-Ozone) [inline i3pystatus/moon.py]: Ah thanks. I fixed your suggestions.

**Pull request merged**